vernac/metasyntax.ml

```diff
@@ -504,27 +507,50 @@ let find_prod_list_loc sfmt fmt =
     (* A separator; we highlight the separating sequence *)
     Loc.merge_opt (fst (List.hd sfmt)) (fst (List.last sfmt))
 
+let is_blank s = List.for_all (fun c -> c = " ") (String.explode s)
```

Comment:
This is a very contrived and inefficient way to write this function. Please write a loop over the string instead.

Reply:
I don't believe that the inefficiency matters here but I made an alternative proposal. Tell me if it is what you intend.